users/views.py

```diff
@@ -288,28 +316,39 @@ def cancel_pending(request, session_id):
 
 @login_required
 def accept_tutee(request, session_id):
-    user = request.user
-    current_user = User.objects.get(id=user.id)
-    chosen_session = Sessions.objects.get(id=session_id)
-    chosen_sessions = Sessions.objects.filter(code=chosen_session.code)
-    tutee = chosen_session.user
-
-    for _session in chosen_sessions:
-        _session.is_accepted = True
-        _session.save()
-
-    chosen_request = Requests.objects.filter(session=chosen_session)
-    for x in chosen_request:
-        x.is_rejected = True
-        x.save()
-
-    new_tutor = current_user
-    new_tutee = tutee
-
-    for _session in chosen_sessions:
-        foo = Sessions_Accepted.objects.create(session=_session, tutor=new_tutor, tutee=new_tutee)
-
-    return redirect('home')
+    if(request.method == "POST"):
+        user = request.user
+        current_user = User.objects.get(id=user.id)
+        chosen_session = Sessions.objects.get(id=session_id)
+        chosen_sessions = Sessions.objects.filter(code=chosen_session.code)
+        tutee = chosen_session.user
+
+        for _session in chosen_sessions:
+            _session.is_accepted = True
+            _session.save()
+
+        chosen_request = Requests.objects.filter(session=chosen_session)
+        for x in chosen_request:
+            x.is_rejected = True
+            x.save()
+
+        new_tutor = current_user
+        new_tutee = tutee
+
+        for _session in chosen_sessions:
+            foo = Sessions_Accepted.objects.create(session=_session, tutor=new_tutor, tutee=new_tutee)
+        return redirect('home')
+    else:
+        current_session = Sessions.objects.get(id = session_id)
+        stringSched = current_session.session_schedule
+        arraySched = stringSched.split()
```

Comment:
For the sake of consistency, would prefer that we go with PEP 8 naming conventions (string_sched instead of stringSched, array_sched instead of stringSched)! Please edit the `mutualSched` view accordingly as well

Reply:
Okie gotcha

Reply:
mutualSched was a test view, it's not relevant

Reply:
Will comment it out nalang